index.js

```diff
@@ -76,7 +76,7 @@ function deleteUnusedImages(options) {
 
     if (unusedImages.length) {
       if (options.delete) {
-        del(unusedImages).then(paths => {
+        del(unusedImages, {force: true}).then(paths => {
```

Comment:
Reluctant to default force delete to true. Is there a reason why you need to force delete images?

Reply:
It is impossible to delete images if the images are not in the directory with node_modules